contracts/router/interfaces/algebra/AlgebraStructs.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.17;
+
+struct ExactInputSingleParams {
+    address tokenIn;
+    address tokenOut;
+    address recipient;
+    uint256 deadline;
+    uint256 amountIn;
+    uint256 amountOutMinimum;
+    uint160 sqrtPriceLimitX96;
+}
+
+struct QuoteExactInputSingleParams {
+    address tokenIn;
+    address tokenOut;
+    uint256 amountIn;
+    uint160 sqrtPriceLimitX96;
```

Comment:
verified contract has this struct param named as `uint160 limitSqrtPrice`. might want to keep that consistent

Reply:
same goes for `ExactInputSingleParams` struct for [router](https://basescan.org/address/0x2dD788D8B399caa4eE92B5492A6A238Fdf2437de#code).

Reply:
Agree, let's be consistent with the contract struct naming

Reply:
It's kind of weird though, as Algebra in-house Router/Quoter contracts seem to be using `limitSqrtPrice`, while the Eden's static quoter is using `sqrtPriceLimitX96` (I guess to keep things consistent among UniV3 forks).

In any case, it feels better to use the Algebra terminology there, as they are the backbone of the integration (even though the quoter we're using uses a slightly different one). Will update in the next commit.